src/profile/stats/stats.container.jsx

```diff
@@ -3,12 +3,26 @@ import Stats from './stats.jsx';
 
 const stateToProps = (state) => ({
 	selectedId: state.list.selectedId,
-	hp: state.profile.stats.hp,
-	attack: state.profile.stats.attack,
-	defense: state.profile.stats.defense,
-	specialAttack: state.profile.stats.specialAttack,
-	specialDefense: state.profile.stats.specialDefense,
-	speed: state.profile.stats.speed,
+	stats: [
+		{ 'name': 'hp',
```

Comment:
I wonder if itd be better to hide this in the reducer

Reply:
Probably, wasn't sure where the best place to hide it was.